jest-playwright.config.js

```diff
@@ -2,14 +2,20 @@ module.exports = {
 
     // Describe which browsers we want to run
 
-    browsers: ["chromium", "firefox", "webkit"],
+    browsers: ["chromium", "webkit", "firefox"],
```

Comment:
Was there a reason for this change? Just curious

Reply:
No, reverted.